specs/workspace_spec.rb

```diff
@@ -0,0 +1,104 @@
+require_relative "test_helper"
+
+describe "Workspace class" do
+  before do
+    VCR.use_cassette("slack_workspace") do
+      @workspace = Workspace.new
+    end
+  end
+
+  it "creates a list of channels and users" do
+    VCR.use_cassette("slack_workspace") do
+      expect(@workspace.channels.first).must_be_kind_of Channel
+      expect(@workspace.users.first).must_be_kind_of User
+    end
+  end
+end
+
+describe "instance methods" do
+  before do
+    VCR.use_cassette("slack_workspace") do
+      @workspace = Workspace.new
+    end
+  end
+
+  it "returns text from print_details" do
+    VCR.use_cassette("slack_workspace") do
+      expect(@workspace.print_details("users")).must_be_kind_of Array
+      expect(@workspace.print_details("channels")).must_be_kind_of Array
+    end
+  end
+
+  it "returns a user object from select_user" do
+    VCR.use_cassette("slack_workspace") do
+      expect(@workspace.select_user("grace.m.shea")).must_be_kind_of User
+      # expect(@workspace.select_user("123")).must_be_kind_of User
```

Comment:
What if you try to select a user that doesn't exist?

What happens to a previously selected user if you try to select a user that D.N.E.?